sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -234,6 +235,9 @@ def __init__(
         memory pressure if you load multiple copies. Given a model that
         consumes N memory and a machine with W cores and M memory, you should
         set this to True if N*W > M.
+      model_copies: The exact number of models that you would like loaded
+        onto your machine. This can be useful if you exactly know your CPU or
```

Comment:
Possible wording suggestion:

This can be useful if you exactly know your CPU or GPU capacity and want to maximize resource utilization. 

> If set, large_model becomes a no-op.

Maybe this should be a ValueError if a user specifies both?

Reply:
> Possible wording suggestion:

Updated

> Maybe this should be a ValueError if a user specifies both?

I don't think it should be a ValueError - if you change it to True and you set this param, that is kinda reasonable and a no-op makes sense IMO since we're still honoring your choice.

Reply:
> but now it also seems that large_model becomes redundant as it is equivalent to passing model_copies = 1, right?

That's right, though I think long term I would like for us to do smart things here (e.g. `large_model` becomes "pack as many models as you can fit). There's some conversation on this general idea in the design doc